distribution/std-lib/Standard/src/Image/Data/Image.enso

```diff
@@ -0,0 +1,111 @@
+from Standard.Base import all
+import Standard.Image.Data.Histogram
+import Standard.Image.Data.Image.Internal
+import Standard.Image.Data.Matrix
+
+polyglot java import org.enso.image.data.Image as Java_Image
+
+type Image
```

Comment:
Do you document anywhere that we use normalised values in the range `[0, 1]`? I've not found it written down anywhere.

Reply:
It's mentioned in the constructor documentation below.